.circleci/config.yml

```diff
@@ -529,6 +529,13 @@ jobs:
     resource_class: gpu.medium
     <<: *pytorch_linux_test_defaults
 
+  pytorch_linux_xenial_py3_test:
+    environment:
+      JOB_BASE_NAME: pytorch-linux-xenial-py3-test
+      PYTHON_VERSION: "3.6"
+    resource_class: large
+    <<: *pytorch_linux_test_defaults
```

Comment:
the test job could be as simple as `python -c "import torch; torch.randn(10).cuda()"; python test/test_jit.py"`. We dont need to run the full test-suite i think.
But if you think it's fully paralllelized and wont add to latency, it's nicer to have full cpu tests run.